### PR TITLE
[3] prepare for v2.0.0

### PR DESCRIPTION
**Pull request by trel**

### CODE REVIEW

**Review by korydraughn — APPROVED** [review]
Looks good.

Pound when ready.

File: setup.py
Comment (alanking) [inline setup.py:39]: Have we tried this on 3.12?

Comment (trel) [reply setup.py:39]: i wrote the new password booth on 3.12.  so, yes!

Comment (trel) [reply setup.py:39]: added

**Review by korydraughn — APPROVED** [review]
Squash/Pound it.

**Comment by trel** [comment]
going to wait until a couple more dan PRs are ready/merged.

**Review by alanking — APPROVED** [review]

**Pull request merged**
